### PR TITLE
New particle effect, generic potion effect, fix for a couple wrong config values

### PR DESCRIPTION
**Pull request by Miirpat-Gaalsien**

### CODE REVIEW

File: MineInAbyss/src/main/java/com/derongan/minecraft/mineinabyss/Ascension/Effect/Effects/VomitAscensionEffect.java [hunk 1]
```diff
@@ -0,0 +1,31 @@
+package com.derongan.minecraft.mineinabyss.Ascension.Effect.Effects;
+
+import net.minecraft.server.v1_12_R1.EnumParticle;
+import net.minecraft.server.v1_12_R1.PacketPlayOutWorldParticles;
+import org.bukkit.Location;
+import org.bukkit.craftbukkit.v1_12_R1.entity.CraftPlayer;
+import org.bukkit.entity.Player;
+
+public class VomitAscensionEffect extends AbstractAscensionEffect {
```
[the remark this answers is not in the document]
Comment (Miirpat-Gaalsien) [reply MineInAbyss/src/main/java/com/derongan/minecraft/mineinabyss/Ascension/Effect/Effects/VomitAscensionEffect.java]: i plan to

File: MineInAbyss/src/main/java/com/derongan/minecraft/mineinabyss/Ascension/Effect/Effects/PotionAscensionEffect.java [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+    }
+
+    //TODO make potion effects be able to merge in different ways
+    /** public void mergeAddPotionEffect(Player player, PotionEffectType potionEffect, int newEffectDuration, int newStrength) {
+             if (player.getPotionEffect(potionEffect).getDuration() < newEffectDuration) {
+                 } else if (player.getPotionEffect(potionEffect).getDuration() > newEffectDuration) {
+                 } else {}
+                 }**/
+
+    public void mergeExtendPotionEffect(Player player, PotionEffectType potionEffect, int newPotionEffectDuration) {
```
[the remark this answers is not in the document]
Comment (Miirpat-Gaalsien) [reply MineInAbyss/src/main/java/com/derongan/minecraft/mineinabyss/Ascension/Effect/Effects/PotionAscensionEffect.java]: shouldn't it be 100 seconds then?
I plan to make the thing also able to merge the strength on effects where it makes a difference.

[the remark this answers is not in the document]
Comment (Derongan) [reply MineInAbyss/src/main/java/com/derongan/minecraft/mineinabyss/Ascension/Effect/Effects/PotionAscensionEffect.java]: Yea I fail at math, but my point stands.

[the remark this answers is not in the document]
Comment (Miirpat-Gaalsien) [reply MineInAbyss/src/main/java/com/derongan/minecraft/mineinabyss/Ascension/Effect/Effects/PotionAscensionEffect.java]: the curse time elapses while the player is ascending, meaning the player has to move really quickly up

**Pull request closed without merging**